cms/djangoapps/api/v1/serializers/course_runs.py

```diff
@@ -125,7 +125,7 @@ def update(self, instance, validated_data):
 class CourseRunSerializerCommonFieldsMixin(serializers.Serializer):
     schedule = CourseRunScheduleSerializer(source='*', required=False)
     pacing_type = CourseRunPacingTypeField(source='self_paced', required=False,
-                                           choices=(('instructor_paced', False), ('self_paced', True),))
+                                           choices=((False, 'instructor_paced'), (True, 'self_paced'),))
```

Comment:
Testing on devstack, this change doesn't seem to affect the visible API.  The conversion between booleans and strings seems to be handled by the CourseRunPacingTypeField methods to_representation and to_internal_value.